package.json

```diff
@@ -28,6 +30,7 @@
     "babel-preset-env": "1.7.0",
     "babel-preset-stage-0": "6.24.1",
     "cssnano": "4.1.10",
+    "http-server": "^0.11.1i",
     "eslint": "5.16.0",
     "eslint-config-airbnb-base": "13.1.0",
     "eslint-config-prettier": "4.2.0",
```

Comment:
Happy for you to consider this out-of-scope if you like, but I think `vanilla-framework` should be in `dependencies` rather than `devDependencies`. It's a dependency in the sense that its contents gets bundled into the built package, right? And if you leave it in `devDependencies` then Renovate will automatically update it, whereas if you put it in `dependencies` it will provide a PR for us to have a chance to review the new version of vanilla.